tests/test_code_execution.py

```diff
@@ -194,7 +194,7 @@ def test_real_generations(sandbox_type):
 @pytest.mark.parametrize(
     "code_begin,code_end,code_output_begin,code_output_end",
     [
-        # ('<llm-code>', '</llm-code>', '<llm-code-output>', '</llm-code-output>'),
+        # ('<llm-code>\n', '</llm-code>\n', '<llm-code-output>\n', '</llm-code-output>\n'),
```

Comment:
Can we enable this test?

Reply:
And I guess add one more for qwen format here and also in the explicit few-shot test below?

Reply:
Right, but it also does extraction of code from those tags. E.g. if you use some special symbol in the tokens that the regex doesn't parse correctly or there is some other bug, it's good if we can catch it here